src/components/Breadcrumbs/Breadcrumbs.stories.mdx

```diff
@@ -113,6 +115,25 @@ denoted by an `aria-current` attribute value of "page".
   </DSProvider>
 </Canvas>
 
+<Canvas>
```

Comment:
I asked for this example before because when the max length was 60 characters, the text `"Grandchild with an Exceptionally Long Name"` was _not_ cut off. But now that the max length is 40 characters, the text _does_ get cut off. So you can remove this second example as both show the same thing.